src/main/java/io/cdap/plugin/successfactors/source/config/SuccessFactorsPluginConfig.java

```diff
@@ -346,20 +366,56 @@ public Builder expandOption(@Nullable String expandOption) {
       return this;
     }
 
+    public Builder authType(@Nullable String authType) {
+      this.authType = authType;
+      return this;
```

Comment:
`authType` will be `null` for existing pipelines.

It should default to `BASIC_AUTH` in that case.